mozci/__init__.py

```diff
@@ -2,7 +2,9 @@
 from pathlib import Path
 
 from adr import sources
-from adr.cli import setup_logging
+
+from mozci.configuration import config  # noqa
```

Comment:
Nit: why the noqa?

Reply:
Flake8 complains about unused imports, but I wanted this to be importable from the root `mozci` module. I guess we could also just instantiate it here rather than in `configuration.py`.